secure-biometric-interface-integrator/lib/standardConstant.js

```diff
@@ -0,0 +1,77 @@
+const states = {
+  LOADING: "LOADING",
+  LOADED: "LOADED",
+  ERROR: "ERROR",
+  AUTHENTICATING: "AUTHENTICATING",
+};
+
+const DeviceStateStatus = {
+  Ready: "READY",
+  "Not Ready": "NOTREADY",
+  Busy: "BUSY",
+  "Not Registered": "NOTREGISTERED",
+};
+
+const DeviceState = {
+  READY: {
+    value: "READY",
+    name: "Ready",
+    class: "ready",
+    symbol: "\u25CF",
+  },
+  NOTREADY: {
+    value: "NOTREADY",
+    name: "Not Ready",
+    class: "not-ready",
+    symbol: "\u25CF",
+  },
+  BUSY: {
+    value: "BUSY",
+    name: "Busy",
+    class: "busy",
+    symbol: "\u25CF",
+  },
+  NOTREGISTERED: {
+    value: "NOTREGISTERED",
+    name: "Not Registered",
+    class: "not-registered",
+    symbol: "\u25CE",
+  },
+};
+
+const DEFAULT_PROPS = {
+  buttonLabel: "scan_and_verify",
+  disable: false,
+  langCode: "en",
+  sbiEnv: {
+    env: "Staging",
+    captureTimeout: 30,
+    irisBioSubtypes: "UNKNOWN",
+    fingerBioSubtypes: "UNKNOWN",
+    faceCaptureCount: 1,
+    faceCaptureScore: 70,
+    fingerCaptureCount: 1,
+    fingerCaptureScore: 70,
+    irisCaptureCount: 1,
+    irisCaptureScore: 70,
+    portRange: "4501-4600",
+    discTimeout: 15,
+    dinfoTimeout: 30,
+    domainUri: `${window.origin}`,
+  },
+  customStyle: {
```

Comment:
Is it possible to introduce a style from the any app which use this library? or they should change this source code?

Reply:
These are the default values, and there is a provision to change these values by the developer with the help of passing prop inputs.

They didn't have to change the source code for this